src/libraries/System.Net.Sockets/tests/FunctionalTests/ArgumentValidationTests.cs

```diff
@@ -800,6 +800,11 @@ public void Connect_ConnectTwice_NotSupported(int invalidatingAction)
         {
             using (Socket client = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp))
             {
+                // Skip on Qemu due to [ActiveIssue(https://github.com/dotnet/runtime/issues/104542)]
+                if (PlatformDetection.IsQemuLinux && invalidatingAction == 1)
+                {
+                    return;
```

Comment:
If you make the method a [ConditionalTheory] instead of [Theory], then here you can throw a new SkipTestException, which will cause this test to be recognized as being skipped rather than passing.